docs/pages/tools/integration/grafana.md

```diff
@@ -0,0 +1,136 @@
+[🏠 Back to HOME](https://awesome-astra.github.io/docs/) | *Last Update {{ git_revision_date }}* 
+
+<img src="../../../../img/grafana/grafana_logo.svg" height="100px" />
+
+## A - Overview
+
+[Grafana](https://grafana.com/) is a multi-platform open source analytics and interactive visualization web application. It provides charts, graphs, and alerts for the web when connected to supported data sources. A licensed Grafana Enterprise version with additional capabilities is also available as a self-hosted installation or an account on the Grafana Labs cloud service. It is expandable through a plug-in system. End users can create complex dashboards using interactive query builders.
+
+Community-developed [Cassandra Datasource for Grafana](https://github.com/HadesArchitect/GrafanaCassandraDatasource) supports both Apache Cassandra as well as DataStax AstraDB, allowing to use Cassandra as a data backend for Grafana. Data can be pulled using simple Qquery Configurator or more advanced but powerful Query Editor.
```

Comment:
typo: simple Query Configurator or more advanced ...

Reply:
Thank you! Fixed.